inference-engine/include/ie_core.hpp

```diff
@@ -79,6 +77,8 @@ class INFERENCE_ENGINE_API_CLASS(Core) {
      * @param model string with model in IR or ONNX format
      * @param weights shared pointer to constant blob with weights
      * ONNX models doesn't support models with data blobs.
+     * If you read ONNX model which requires external data feature,
+     * use ReadNetwork overloading which takes path to the model.
```

Comment:
can we refer to exact overloading? E.g. `use InferenceEngine::Core::ReadNetwork(const std::string& model, const Blob::CPtr& weights) const`